cbor/event.go

```diff
@@ -47,14 +46,14 @@ func NewEvent(ctx context.Context, dag format.DAGService, body format.Node, sett
 		return nil, err
 	}
 	eventHeader := &eventHeader{
-		Time: settings.Time.Unix(),
+		Time: time.Now().Unix(),
```

Comment:
Should we think about getting an HCL implementation in here soon? Something like https://github.com/lafikl/hlc looks very simple?

Reply:
Nice—issue added: https://github.com/textileio/go-textile-threads/issues/76